common/config/rush/command-line.json

```diff
@@ -168,6 +168,54 @@
     // }
   ],
 
+  /* PHASES */
+  "phases": [

```

Comment:
How are these phases mapped to Rush actions?  I think we're missing the definition in `"commands"`.

Reply:
If this is a realistic example, we should show the intended Rush shell commands for realistic everyday tasks:

- How do I build everything including all phases?
- How do I update the README and do nothing else?
- How do I do a `--lite` build where we skip unit tests and linting?
- For these things, what's the equivalent of `rush rebuild`?

Maybe the answer is that initially each of these problems requires a completely new command action name.  (`rush build-lite`?)  But if so we can still enumerate them to get a sense of how painful that feature cut will be in practice.

Reply:
These are great questions.

Since presumably the monorepo had this `--lite` need before phases, I implemented a solution in `main` (a custom build script that lets it properly handle `--lite` from the rushx command line).

That `--lite` flag is rough for the phased approach though.  As an example, I imagined that the user wanted `build.js` to pass `--lite` both to Heft and to the `push_notes.js` custom script.  If I invent a new phased command `build-lite`, then it can have phases `['_phase:compile', '_phase:readme_updater']` etc., and you don't need to pass `--lite` to Heft because you've baked that into the phase. But that's not generically possible if you were doing something with the `--lite` flag.

I guess one possible approach is to implement lite _phases_, `push-notes` and `push-notes-lite`, and only include the appropriate one.  But they'll never share build cache, and would have to be leaf nodes (no dependents), otherwise how would a dependent phase know which one to rely on.